apps/extension/src/routes/popup/home/index.tsx

```diff
@@ -34,31 +33,23 @@ export const popupIndexLoader = async (): Promise<Response | PopupLoaderData> =>
 
 export const PopupIndex = () => {
   const account = useStore(selectedAccount);
-  const { next, previous, setIndex } = useStore(accountsSelector);
+  const { ephemeral, next, previous, setIndex, setEphemeral } = useStore(accountsSelector);
 
   return (
     <div className='relative flex h-full flex-col items-stretch justify-start bg-left-bottom px-[30px]'>
       <div className='absolute bottom-[50px] left-[-10px] -z-10 h-[715px] w-[900px] overflow-hidden bg-logo opacity-10' />
       <IndexHeader />
-      <div className='my-32 flex w-full flex-col'>
-        <SelectAccount previous={previous} next={next} setIndex={setIndex} index={account?.index} />
-        <div className='mt-4 flex items-center justify-between gap-1 break-all rounded-lg border bg-background px-3 py-4'>
-          <div className='flex items-center gap-[6px]'>
-            <Identicon name={account?.address ?? ''} className='h-6 w-6 rounded-full' />
-            <p className='select-none text-center font-mono text-[12px] leading-[18px] text-muted-foreground'>
-              {account?.preview}
-            </p>
-          </div>
-          <CopyToClipboard
-            text={account?.address ?? ''}
-            label={
-              <div>
-                <CopyIcon className='h-4 w-4 text-muted-foreground hover:opacity-50' />
-              </div>
-            }
-            className='w-4'
+      <div className='my-32'>
+        {account && (
+          <SelectAccount
+            previous={previous}
+            next={next}
+            setIndex={setIndex}
+            account={account}
+            ephemeral={ephemeral}
+            setEphemeral={setEphemeral}
```

Comment:
In the popup, upon every block that gets synced, it appears the ephemeral address gets refreshed. Is that on purpose? Think it should stay the same between blocks.